src/component/canvas/ScatterRenderable.jsx

```diff
@@ -111,6 +111,8 @@ export class ScatterRenderable extends React.PureComponent {
         index: number
     ): ?React.Element<any> {
         if(row.x == null || row.y == null) return null;
+        if(isNaN(row.x) || isNaN(row.y)) return null;
```

Comment:
I'd prefer to not add lodash as a dependency, I think seeing as these renderables are really only used internally we can be pretty explicit about what they accept so don't need to handle the edge cases that lodash's function would. The line length of:

```
if(typeof row.x !== 'number' || typeof row.y !== 'number') return null;
```
isn't too bad I don't think. We generally try to stay under 100 chars.